app/controllers/IndexController.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use Phalcon\Mvc\Controller;
+
+class IndexController extends Controller
+{
+
+    public function indexAction()
+    {
+        echo "<h1>Hello!</h1>";
```

Comment:
I understand this is basic setup but make sure controllers never explicitly echo data.